Example/package.json

```diff
@@ -7,6 +7,7 @@
     },
     "dependencies": {
         "react-native": "^0.20.0",
-        "react-native-root-toast": "^0.0.1"
+        "react-native-root-toast": "../",
```

Comment:
Hi, I keep seeing this structure elsewhere and I guess it may be only a convenience for contributors to also test the example project after they did any changes to the lib (needs npm uninstall this lib and re npm install), because they won't have an updated npm module of this lib to npm install before it's merged and officially published; 
and a normal lib user would most possibly clone the whole repository and install from within the example subfolder, while he should also know how to change the version if he want the example to be installed independently cause it's very basic.
Anyway I am still a learner of RN, please feel free to take any changes you like :)